doc/conf.py

```diff
@@ -62,6 +62,10 @@
 # The base URL with a proper language and version.
 html_baseurl = os.environ.get("READTHEDOCS_CANONICAL_URL", "/")
 
+# Tell Jinja2 templates the build is running on Read the Docs
+if os.environ.get("READTHEDOCS", "") == "True":
+    html_context["READTHEDOCS"] = True
+
```

Comment:
Based on the [build error](https://readthedocs.com/projects/xanaduai-pennylane/builds/2353263/), looks like we might just need to define an empty `html_context` dictionary?